src/main/java/io/ipfs/api/IPFS.java

```diff
@@ -67,10 +67,18 @@ public IPFS(MultiAddress addr) {
     }
 
     public IPFS(String host, int port, String version, boolean ssl) {
-        this(host, port, version, DEFAULT_CONNECT_TIMEOUT_MILLIS, DEFAULT_READ_TIMEOUT_MILLIS, ssl);
+        this(host, port, version, false, DEFAULT_CONNECT_TIMEOUT_MILLIS, DEFAULT_READ_TIMEOUT_MILLIS, ssl);
+    }
+
+    public IPFS(String host, int port, String version, boolean overrideMinVersionCheck, boolean ssl) {
```

Comment:
Can we make it positive rather than negative. So enforceMinVersion?

Reply:
done